jcl/src/java.base/share/classes/com/ibm/tools/attach/target/AttachHandler.java

```diff
@@ -102,6 +102,12 @@
 	private static String nameProperty;
 	private static String pidProperty;
 	private static int numberOfTargets;
+	/*[IF Sidecar19-SE]*/
+	/*[PR 132828 don't allow VM to attach to itself unless explicitly enabled ]*/
```

Comment:
This is likely an internal ticket number - can you remove it and add any necessary context to the code or the PR?

Reply:
Fixed.